src/main/java/org/ibp/api/rest/labelprinting/ObservationDatasetLabelPrinting.java

```diff
@@ -281,7 +281,7 @@ public LabelsData getLabelsData(final LabelsGeneratorInput labelsGeneratorInput)
 		studyTransactionsRequest.setTransactionsSearch(transactionsSearch);
 
 		final List<StudyTransactionsDto> studyTransactionsDtos =
-			studyTransactionsService.searchStudyTransactions(labelsGeneratorInput.getStudyId(), studyTransactionsRequest);
+			studyTransactionsService.searchStudyTransactions(labelsGeneratorInput.getStudyId(), studyTransactionsRequest, null);
```

Comment:
How about converting the `Pageable` parameter as `Optional<Pageable>`? That way we wouldn't be passing decreasing null values as parameters if paging is not applicable.